### ISSUE TITLE
[Feature] Add Single-URL cache purging

### ISSUE THREAD

**Issue by marklagendijk** [zingz0r/CloudFlare.Client#94]
**Requested CloudFlare API endpoint**
POST /zones/{zone_id}/purge_cache

**Documentation Url to requested endpoint**
https://developers.cloudflare.com/api/operations/zone-purge

**Additional context**
The endpoint is called 'Single-URL cache purging', but actually allows specifying multiple urls.
The documentation does not describe the body of doing url-based cache purging.
However, based on searching on the forum I believe both of these are valid:

Simple variant specifying url strings directly:
```json
{
  "files": [
    "https://www.example.com/path/mypage1",
    "https://www.example.com/path/mypage2"
  ]
}
```

Variant which specifies headers:
```json
{
  "files": [
    {
      "url": "https://www.example.com/path/mypage1",
      "headers": {
        "Origin": "https://www.example2.com"
      }
    },
    {
      "url": "https://www.example.com/path/mypage2",
      "headers": {
        "Origin": "https://www.example2.com"
      }
    }
  ]
}
```

I would propose implementing the 2nd variant, because that should work for all use-cases.

**Comment by marklagendijk** [comment]
The PR is ready for review / merging.

**Comment by zingz0r** [comment]
Thanks! I've merged it. It will be available in the next version 8.0.0
